compiler/src/dotty/tools/dotc/core/Names.scala

```diff
@@ -301,6 +305,9 @@ object Names {
         new String(chrs, start, length)
       }
 
+    /** It's OK to take a toString if the stacktrace does not occur a method
```

Comment:
typo: "a method" -> "in a method"
And on the line below: "or it also" -> "or if it also"